lib/faraday/log_filter/middleware.rb

```diff
@@ -0,0 +1,59 @@
+require "faraday"
+require 'forwardable'
+
+module Faraday
+  module LogFilter
+    class Middleware < Faraday::Response::Middleware
```

Comment:
Good call out on the `super` at the bottom.

May have to directly invoke `call` on `@app` like:

```ruby
@app.call(env).on_complete do |environment|
  on_complete(environment)
end
```